index.js

```diff
@@ -0,0 +1,127 @@
+module.exports = {
+    "plugins": [
+        "import"
+    ],
+    "extends": "eslint:recommended",
+    "env": {
+        "browser": true,
+        "es6": true
```

Comment:
Should we split this `env` to an `incuna-jsnext`? I'd like the base `incuna` be able to be used in any JS environment, not just browser and es6.

Or, considering our coding style doesn't change between environments, we don't need this `env`: it can be set in the projects based on the source files.